docs/dev/guidelines/client.rst

```diff
@@ -247,4 +251,54 @@ Do not use ``<span [ngbTooltip]="submittedDate | artemisDate">{{ submittedDate |
 
 Use ``<span [ngbTooltip]="submittedDate | artemisDate">{{ submittedDate | artemisTimeAgo }}</span>``
 
+14. Chart Instantiation
+=======================
+
+We are using the framework ngx-charts in order to instantiate charts and diagrams in Artemis.
+
+The following is an example HTML template for a vertical bar chart:
+
+.. code-block:: html+ng2
+
+    <div #containerRef class="col-md-9">
+        <ngx-charts-bar-vertical
+            [view]="[containerRef.offsetWidth, 300]"
+            [results]="ngxData"
+            [scheme]="color"
+            [legend]="false"
+            [xAxis]="true"
+            [yAxis]="true"
+            [yScaleMax]="20"
+            [roundEdges]="true"
+            [showDataLabel]="true">
+            <ng-template #tooltipTemplate let-model="model">
+                {{ labelTitle }}: {{ roundValue((model.value / totalValue) * 100) }}%
```

Comment:
I think we need to come up with the standardised way of rounding values. I learned recently from @lschlesinger that we have a `round` function defined in utils: https://github.com/ls1intum/Artemis/blob/develop/src/main/webapp/app/shared/util/utils.ts#L61-L80

I think it's worth mentioning.

Reply:
Thanks for the hint, I will change this immediately.